scripts/ci/jenkins_pipeline_csi

```diff
@@ -1,4 +1,12 @@
 pipeline {
+    environment {
+        registryCredential = 'csi_w3_user'
+        artifactory_path = 'build/reports/images_url'
+        registry_protocol = 'https'
+        Dockerfile_controller = 'Dockerfile-csi-controller'
+        Dockerfile_node = 'Dockerfile-csi-node'
+        latest_tag = "develop origin/develop master"
```

Comment:
why not just keep a boolean var (like the original code)?
i.e. something like `latest_tag = "develop origin/develop master".contains(GIT_BRANCH)`

Reply:
I didn't find a way to do it in Jenkins

Reply:
what do you mean by "find"? what's wrong with my suggestion?
and as I wrote, if this value remains, then at least the name should be changed (to e.g. `latest_tag_branches`).

Reply:
done